src/app/patient-summary/components/resources/Patient/index.tsx

```diff
@@ -11,54 +12,67 @@ const rows: TRow<TPatient>[] = [
   {
     type: 'row',
     config: {
-      field: 'name',
       label: 'Name',
-      renderRow: (field) =>
-        `${field['name']?.[0].given[0]}, ${field['name']?.[0].family}`,
+      render: ({ name }) =>
+        [name?.[0].given?.flat().join(' '), name?.[0].family]
+          .filter((name) => !!name)
+          .join(', '),
     },
   },
   {
     type: 'row',
     config: { field: 'gender', label: 'Gender' },
   },
   { type: 'row', config: { field: 'birthDate', label: 'Birth Date' } },
+  {
+    type: 'row',
+    config: {
+      field: 'maritalStatus',
+      label: 'Marital Status',
+      render: ({ maritalStatus }) => maritalStatus?.coding?.[0].display,
+    },
+  },
   {
     type: 'table',
     config: {
       title: 'Patient Identifiers',
       columns: ['Identifier', 'System'],
-      renderRow: ({ row, StyledTableRow, StyledTableCell }) =>
-        row.identifier?.map((data, index) => (
-          <StyledTableRow key={`${JSON.stringify(data)}_${index}`}>
-            <StyledTableCell>{data.value}</StyledTableCell>
-            <StyledTableCell>{data.system}</StyledTableCell>
+      render: ({ row, StyledTableRow, StyledTableCell }) =>
+        row.identifier?.map(({ value, system, type }) => (
+          <StyledTableRow key={uuid()}>
+            <StyledTableCell>{value}</StyledTableCell>
+            <StyledTableCell>
+              {system}
+              {type?.coding?.[0]?.system}
+            </StyledTableCell>
           </StyledTableRow>
-        )),
-      resource: (datum) => datum,
+        )) || [],
     },
   },
   {
     type: 'table',
     config: {
       title: 'Connection Details',
       columns: ['Type', 'Info'],
-      renderRow: ({ row, StyledTableRow, StyledTableCell }) => [
-        row.address?.map((data, index) => (
-          <StyledTableRow key={`${JSON.stringify(data)}_${index}`}>
-            <StyledTableCell>Address</StyledTableCell>
-            <StyledTableCell>{`${data.line},${data.city}, ${data.postalCode}, ${data.country}`}</StyledTableCell>
+      render: ({ row, StyledTableRow, StyledTableCell }) => [
+        row.address?.map(({ use, line, city, postalCode, country, type }) => (
+          <StyledTableRow key={uuid()}>
+            <StyledTableCell>
+              {camelCaseToFlat(use ? `${use} address` : 'address')}
+              {type && ` (${type})`}
+            </StyledTableCell>
+            <StyledTableCell>{`${line}, ${city}, ${postalCode}, ${country}`}</StyledTableCell>
```

Comment:
Hello @yassinedorbozgithub , I would suggest this format, to avoid dealing with null values, in case one of the values wasn't present.
```suggestion
            <StyledTableCell>{[line, city, postalCode, country].filter(Boolean).join(', ')}</StyledTableCell>
```